admin/client/src/components/PopoverField/PopoverField.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { Popover, OverlayTrigger } from 'react-bootstrap-4';
+import SilverStripeComponent from 'lib/SilverStripeComponent';
+
+class PopoverField extends SilverStripeComponent {
+
+  render() {
+    const placement = this.getPlacement();
+    const overlay = (
+      <Popover id={`${this.props.id}_Popover`} className={`fade in popover-${placement}`}
+        title={this.getPopoverTitle()}
+      >
+        {this.props.children}
+      </Popover>
+    );
+    // If no text is specified, use ... with xl icon style
+    const buttonClasses = ['btn', 'btn-secondary', 'btn--no-focus'];
+    if (!this.props.title) {
```

Comment:
Title is title of the button, popover title is the title of the popup window. You can have one and not the other, so it shouldn't inherit from either.